ote/ote/api.py

```diff
@@ -105,6 +105,7 @@ def export_args_parser(template_path):
                             help='Location where exported model will be stored.')
         parser.add_argument('--onnx', action='store_true', default=config['output_format']['onnx']['default'],
                             help='Enable onnx export.')
+        parser.add_argument('--opset', type=int, default=10, help='ONNX opset')
```

Comment:
it should not be here, for now at least.

Does opset=11 not work for everything?